helpers/result_to_json.cr

```diff
@@ -57,13 +58,18 @@ def convert_to_test_cases(test_suite : XML::Node)
         next nil
       end
 
+      error = find_element(test_case, TAG_ERROR)
       failure = find_element(test_case, TAG_FAILURE)
       skipped = find_element(test_case, TAG_SKIPPED)
 
-      status = failure ? FAIL : (skipped ? ERROR : PASS)
-      message = failure ? failure.not_nil![ATTR_MESSAGE] : (
-        skipped ? "Test case unexpectedly skipped" : nil
-      )
+      status = (error || failure) ? FAIL : (skipped ? ERROR : PASS)
```

Comment:
Was there any particular reason you don't use the `error` status for when `error` is set? Or is error also used when a test fails (instead of errors)?

Reply:
Updated to obtain the test case status using `case` block rather than chained ternary